subt/artifacts.py

```diff
@@ -97,8 +97,9 @@ def count_red(img, filtered=False, stdout=None):
         mask = b == 255
         reddish = b.astype(np.uint8)
         __, bin_img = cv2.threshold(reddish, 160, 255, cv2.THRESH_BINARY)
-        
-        im2, contours, hierarchy = cv2.findContours(bin_img, cv2.RETR_EXTERNAL, cv2.CHAIN_APPROX_NONE)
+
+        # cv2.findCountours for opencv == 3 returns 3 values while for opencv == 4 returns just 2
+        contours, hierarchy = cv2.findContours(bin_img, cv2.RETR_EXTERNAL, cv2.CHAIN_APPROX_NONE)[-2:]
```

Comment:
OK, so this code will crash for OpenCV 3.x right?

Reply:
No. Why would it? I took especial care to have it working for both version returning either 2 or 3 arguments. That is why there is the `[-2:0]` - to take the last 2 values returned (the first value was dropped going to OpenCV 4).